robot_ws/src/martian_detector/nodes/notifier

```diff
@@ -13,63 +13,66 @@
  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
  SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 """
-
+import datetime as dt
 import rospy
 import boto3
 from std_msgs.msg import String
 from std_msgs.msg import Int64MultiArray
 
-aws_region = rospy.get_param('~/notifier/aws_region', 'us-west-2')
-s3_bucket_name = rospy.get_param('~/notifier/s3_bucket_name')
-search_object = rospy.get_param('~/notifier/search_object', 'Alien')
-phone_number = rospy.get_param('~/notifier/phone_number', '')
 
-s3 = boto3.resource('s3', aws_region)
-sns = boto3.client('sns', aws_region)
+DETECTIONS_PREFIX = 'detections'
+# sns = boto3.client('sns', aws_region)  #  Disable SNS for now, may enable it later if needed
```

Comment:
This was a interesting piece of the workshop. Would be good to have it back in some flavour! Doesn't have to to be SNS directly, could be IoT Events > SNS or something like that. I agree though, I think the S3 file + dashboard is probably sufficient for the first revision of this workshop.

Reply:
that's the reason I didn't delete it. It's nice to have a "visual" way of showing a detection but SMS is probably an overkill and it didn't work anyways. Happy to add alternatives, let's talk about it